either/index.ts

```diff
@@ -21,6 +21,7 @@ function isWrappedFunction<A, B, L>(
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 type StaticCheck = ClassImplements<
   typeof EitherConstructor,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
How do you think, could  it make sense to add the rule as a global project eslint rule? There are a lot of places like this, so, I think it's not so clean to have a lot of those kinds of comments.